File: Basics_with_Python/PythonBasicCourse/2_Conditional_Statements/6_Conditional_Statements_Excercises___Exercise/more_exercises/07._Fuel_Tank.py
fuel_type = input()
fuel_left = float(input())
fuel_type = fuel_type.lower()
if fuel_type == "diesel":
    if fuel_left >= 25:
        print(f"You have enough {fuel_type}.")
    else:
        print(f"Fill your tank with {fuel_type}!")
elif fuel_type == "gasoline":
    if fuel_left >= 25:
        print(f"You have enough {fuel_type}.")
    else:
        print(f"Fill your tank with {fuel_type}!")
elif fuel_type == "gas":
    if fuel_left >= 25:
        print(f"You have enough {fuel_type}.")
    else:
        print(f"Fill your tank with {fuel_type}!")
else:
    print("Invalid fuel!")

File: Basics_with_JavaScript/56_Conditional_Statements/More_Exersises/07._Fuel_Tank.js
function vladi(input){

    let fuel = String(input[0]);
    let litersInTank = Number(input[1]);
    fuel = fuel.toLowerCase()

    if (fuel === "diesel" || fuel === "gasoline" || fuel === "gas") {
        if (litersInTank >= 25) {
            console.log(`You have enough ${fuel}.`);
        } else {
            console.log(`Fill your tank with ${fuel}!`);
        }
    } else if (fuel !== "diesel" && fuel !== "gasoline" && fuel !== "gas") {
        console.log(`Invalid fuel!`);
    }

    
    }
   
    // vladi(["Diesel", 10]);
    vladi(["Gasoline", 40]);
    vladi(["Gas", 25]);
    vladi(["Kerosine", 200]);